### PR TITLE
[#3862] improvement(docker-image): Start multiple HMSs in Kerberos Hive docker image

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Add another Hive metastore server instance in the Hive docker iamge.

### Why are the changes needed?

We are going to check if Gravitino supports Kerberos authentication with multiple HMS, so starting another HMS in the existing Kerberos Hive docker image is in great need.

Fix: #3862 
Fix: #4240

### Does this PR introduce _any_ user-facing change?

N/A.

### How was this patch tested?

N/A.

### CODE REVIEW

File: docs/docker-image-details.md [hunk 1]
```diff
@@ -96,6 +96,13 @@ You can use these kinds of Docker images to facilitate integration testing of al
 You can use this kind of image to test the catalog of Apache Hive with kerberos enable
 
 Changelog
+- gravitino-ci-kerberos-hive:0.1.5
+  - Start another HMS service in the container with port 19083.
```
Comment (mchades) [inline docs/docker-image-details.md:100]: Please describe the differences between this HMS and another one, as well as their use cases.

File: docs/docker-image-details.md [hunk 2]
```diff
@@ -96,6 +96,13 @@ You can use these kinds of Docker images to facilitate integration testing of al
 You can use this kind of image to test the catalog of Apache Hive with kerberos enable
 
 Changelog
+- gravitino-ci-kerberos-hive:0.1.5
```
Comment (mchades) [inline docs/docker-image-details.md:99]: Did you forget to update the gradle.kts to enable the image?

**Review by mchades — COMMENTED** [review]
Suggest putting the test code in this PR as well to verify the correctness of the image configuration.

File: docs/docker-image-details.md (hunk 1 above)
Comment (yuqi1129) [reply docs/docker-image-details.md]: done

**Comment by yuqi1129** [comment]
@mchades 
Please help to take a look again

File: dev/docker/kerberos-hive/hive-site1.xml [hunk 3]
```diff
@@ -0,0 +1,77 @@
+<configuration>
+  <property>
+    <name>hive.server2.enable.doAs</name>
+    <value>true</value>
+    <description>Disable user impersonation for HiveServer2</description>
```
Comment (mchades) [inline dev/docker/kerberos-hive/hive-site1.xml:5]: enable or disable?

**Review by mchades — APPROVED** [review]
Overall LGTM

File: dev/docker/kerberos-hive/hive-site1.xml (hunk 3 above)
Comment (yuqi1129) [reply dev/docker/kerberos-hive/hive-site1.xml:5]: It's a mistake that exists for a long time, I copied the content from the Hive docker file.

Comment (yuqi1129) [reply dev/docker/kerberos-hive/hive-site1.xml]: I fixed the mistake for Kerberos Hive docker fille, as for Hive without Kerberos, I believe we need to use another PR to fix it or fix it by the way.

Comment (mchades) [reply dev/docker/kerberos-hive/hive-site1.xml]: for Hive without Kerberos can be fixed in another image updating

**Review by mchades — APPROVED** [review]
LGTM

**Comment by mchades** [comment]
@yuqi1129 don't forget to check out and push a new branch for new image

**Pull request merged**